### PR TITLE
[chip dv] Coverage exclusions and connectivity test additions

### PR DESCRIPTION
**Pull request by sriyerg**
What the title says- please see individual commit descriptions for details.

### CODE REVIEW

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 1]
```diff
@@ -821,7 +821,24 @@
...
+              integrity error on the command channel.
+            - Verify that the fatal alert fired on the PLIC output.
+            - Reboot the chip and this time, inject a fatal alert through violation of the reg
+              write-enable one hot check using the standardized sec_cm_pkg framework.
+            - Verify that the fatal alert fired on the PLIC output.
+            '''
+      stage: V3
+      tests: []
+    }
+[UNR] The le_i input is tied off to 0 (only level interrupts supported).
```
Comment (tjaychen) [inline hw/top_earlgrey/data/chip_testplan.hjson:841]: is this UNR note supposed to be here?

File: hw/top_earlgrey/dv/cov/spi_host_device_unr_tied_off.el [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: root
+// Format Version: 2
+// Date: Fri Nov 11 20:59:05 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "2715753694 4000582234"
```
Comment (tjaychen) [inline hw/top_earlgrey/dv/cov/spi_host_device_unr_tied_off.el:11]: @eunchan 
@weicaiyang to double check, but this part looks good to me.

File: hw/top_earlgrey/dv/cov/plic_le_i_tied_off.el [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: root
+// Format Version: 2
+// Date: Fri Nov 11 23:31:21 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "3217127264 1123521396"
+INSTANCE: tb.dut.top_earlgrey.u_rv_plic.u_gateway
+ANNOTATION: "[UNR] The le_i input is tied off to 0 (only level interrupts supported)."
```
Comment (tjaychen) [inline hw/top_earlgrey/dv/cov/plic_le_i_tied_off.el:13]: ah, i had not realized we completely disabled the edge function.

File: hw/top_earlgrey/dv/cov/conn_otp_ctrl_ast_obs_ext_volt.el [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+// Date: Fri Nov 11 23:06:36 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "1202717726 1014860158"
+INSTANCE: tb.dut.top_earlgrey
+ANNOTATION: "[LOW_RISK] Covered via connectivity. Cannot be covered in DV due to behavioral models of AST and OTP."
+Toggle otp_ext_voltage_h_io "net otp_ext_voltage_h_io"
+CHECKSUM: "708483626 3106053973"
+INSTANCE: tb.dut.top_earlgrey.u_otp_ctrl
+ANNOTATION: "[LOW_RISK] Covered via connectivity. Cannot be covered in DV due to behavioral models of AST and OTP."
```
Comment (tjaychen) [inline hw/top_earlgrey/dv/cov/conn_otp_ctrl_ast_obs_ext_volt.el:17]: should we say.. "must be covered in vendor closed source DV"?

**Review by tjaychen — APPROVED** [review]
looks good to me with one question.
it would be good if others can have a look also.

File: hw/formal/tools/jaspergold/jaspergold.hjson [hunk 5]
```diff
@@ -7,6 +7,9 @@
                "-proj jgproject",
                "-allow_unsupported_OS"]
 
+  // If this line is not seen in the log, then fail the test.
+  build_pass_patterns: ["^INFO: Proof threads stopped\\.$"]
+
```
Comment (cindychip) [inline hw/formal/tools/jaspergold/jaspergold.hjson:12]: Thanks Sri!

**Review by cindychip — APPROVED** [review]
I checked chip_cover, plic, and otp related exclusion files. LGTM

File: hw/top_earlgrey/dv/cov/spi_host_device_unr_tied_off.el (hunk 2 above)
Comment (weicaiyang) [reply hw/top_earlgrey/dv/cov/spi_host_device_unr_tied_off.el:11]: This also LGTM

**Review by weicaiyang — APPROVED** [review]
spi_device, spi_host portion LGTM

File: hw/top_earlgrey/data/chip_testplan.hjson (hunk 1 above)
Comment (sriyerg) [reply hw/top_earlgrey/data/chip_testplan.hjson]: oops, removed..

File: hw/top_earlgrey/dv/cov/conn_otp_ctrl_ast_obs_ext_volt.el (hunk 4 above)
Comment (sriyerg) [reply hw/top_earlgrey/dv/cov/conn_otp_ctrl_ast_obs_ext_volt.el]: Sure, done.

**Pull request merged**